lib/takeaway.rb

```diff
@@ -0,0 +1,73 @@
+require 'twilio-ruby'
+
+class Order
+attr_reader :menu, :order, :total, :total_items, :meal_choice, :meal_count, :total_cost
+
+def initialize
+  @menu = {Pizza: 10, Pasta: 7.50, Cheesesteak: 8.50, Fries: 3.50, Donut: 2.50}
```

Comment:
can you make it such that the menu isn't hard-coded into the program, eg by using a Menu class?

Reply:
> For example the menu items should not be hard coded into a restaurant class. Arguably they should not be in the business logic at all, e.g. being added at runtime (i.e. in IRB) or loaded from an external hash or maybe even a file. We are concerned that the menu items will likely change